packages/amplify_core/lib/src/types/plugin/amplify_plugin_interface.dart

```diff
@@ -30,7 +30,7 @@ abstract class AmplifyPluginInterface {
   Future<void> addPlugin() async {}
 
   /// Configures the plugin using the registered [config].
-  Future<void> configure({AmplifyConfig? config}) async {}
+  Future<void> configure({String? config}) async {}

```

Comment:
Let's change it back and convert to a string in the DataStore configure method. This is an internal API so it's okay to break it, but every Dart plugin will expect a typed config.

Reply:
After discussion, I'm removing this entirely.

Reply:
After discussion, I am removing all implementations of configure from datastore category/plugin hierarchy.